packages/grid/test/grid-wrapper.common.js

```diff
@@ -0,0 +1,59 @@
+import { expect } from '@esm-bundle/chai';
+import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
+import { flushGrid, getBodyCellContent } from './helpers';
```

Comment:
nit: missing .js suffix. If you use VScode, [this](https://stackoverflow.com/questions/62503006/vscode-add-js-extension-on-import-autocomplete/62503247#62503247) is a good-to-have setting.

Reply:
Done.